RELEASE-NOTES.md

```diff
@@ -0,0 +1,15 @@
+# v2.0.0
```

Comment:
The problem with argument (_1) i what i was dealing with in #374 - some of the python types get mapped to sql types, some do not.  So you have to declare str, and then it turns to varchar, but in case of list - you have to declare array from the beggining.

Reply:
The issue _1 is about wrong example of identifier: `hello(int, string)` - it cannot work because `create` requires also argument names, so IMO the example should like `hello(name string)`.

Reply:
_1) This is ok, I have to fix helps